ManifoldTests/InferenceTests.swift

```diff
@@ -2,21 +2,24 @@
 
 final class InferenceTests: XCTestCase {
 	func testVariablesAreAssignedAFreshTypeVariable() {
-		assertEqual(infer(0).0, Type(Variable()))
+		assertEqual(infer(0).0.isVariable, true)
 	}
 
 	func testApplicationsAreAssignedAFreshTypeVariable() {
 		let application = 0 <| 0 // fixme: left operand should be an abstraction
-		assertEqual(infer(application).0, Type(Variable()))
+		assertEqual(infer(application).0.isVariable, true)
 	}
 
 	func testAbstractionsAreAssignedAFunctionType() {
 		let abstraction = 0 .. 0
-		assertEqual(infer(abstraction).0, Type(Variable()) --> Type(Variable()))
+		assertEqual(infer(abstraction).0.isFunction, true)
```

Comment:
Since type variable equality is no longer defined only up to renaming, we test with classification properties.